app/scripts/migrations/092.3.test.ts

```diff
@@ -0,0 +1,132 @@
+import { migrate } from './092.3';
+
+const PREFERENCES_CONTROLLER_MOCK = {
+  useBlockie: false,
+  useNonceField: false,
+  usePhishDetect: true,
+  dismissSeedBackUpReminder: false,
+  disabledRpcMethodPreferences: {
+    eth_sign: false,
+  },
+  useMultiAccountBalanceChecker: true,
+  useTokenDetection: false,
+  useNftDetection: false,
+  use4ByteResolution: true,
+  useCurrencyRateCheck: true,
+  openSeaEnabled: false,
+  advancedGasFee: null,
+  featureFlags: {
+    showIncomingTransactions: true,
+  },
+  knownMethodData: {},
+  currentLocale: 'EN',
+  identities: {},
+  lostIdentities: {},
+  forgottenPassword: false,
+  preferences: {
+    autoLockTimeLimit: undefined,
+    showFiatInTestnets: false,
+    showTestNetworks: false,
+    useNativeCurrencyAsPrimaryCurrency: true,
+    hideZeroBalanceTokens: false,
+  },
+  // ENS decentralized website resolution
+  ipfsGateway: '',
+  useAddressBarEnsResolution: true,
+  infuraBlocked: null,
+  ledgerTransportType: 'U2F',
+  snapRegistryList: {},
+  transactionSecurityCheckEnabled: false,
+  theme: 'OS',
+  isLineaMainnetReleased: false,
+};
+
+const PREFERENCES_CONTROLLER_MOCK_2 = {
```

Comment:
Nit: Maybe it would be better to inline these added properties? So that it's more clear in each test what the start conditions are

Reply:
Done.